corehq/apps/app_manager/detail_screen.py

```diff
@@ -367,16 +367,17 @@ class Enum(FormattedDetailColumn):
     def _make_xpath(self, type):
         return sx.XPathEnum.build(
             enum=self.column.enum,
+            type=type,
```

Comment:
Instead of turning the return value of `_xpath_template` into a tuple, I'd add another parameter here for the format. `template` is a concept used throughout this file, so it's not ideal for it to have multiple meanings depending on which kind of `FormattedDetailColumn` you're dealing with.

The format strings you're passing should be available here as `self.slug`, which gets populated by the `register_format_type` decorator: https://github.com/dimagi/commcare-hq/blob/31c23745e8ec5014c1b764d5094a013dac64906b/corehq/apps/app_manager/detail_screen.py#L42-L45